src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function (valueResult) {
+      return function (value) {
+        return valueResult + value;
+      };
+    },
+
+    subtract: function (valueResult) {
+      return function (value) {
+        return valueResult - value;
+      };
+    },
+
+    multiply: function (valueResult) {
+      return function (value) {
+        return valueResult * value;
+      };
+    },
```

Comment:
[CODE KNOWLEDGE] - Use the method shorthand syntax for the `multiply` method as well.